bones/exemplar.go

```diff
@@ -228,13 +228,16 @@ func (f *formatter) FieldExemplar(fd protoreflect.FieldDescriptor) exemplar {
 // The value is as per the exemplar form for the fields type. See the other
 // *Exemplar functions for details of those.
 //
-// Map fields are emitted as repeated key/value message pairs in the expanded
-// backward-compatible form, as opposed to {key: value, ...} objects.
+// Map fields are emitted as objects with non-string keys converted to strings.
 //
 // Repeated fields are emitted with a single element exemplar of the repeated
 // type.
 func (f *formatter) FieldValueExemplar(fd protoreflect.FieldDescriptor) exemplar {
 	var e exemplar
+	if fd.IsMap() {
+		return f.MapExemplar(fd)
```

Comment:
maybe add a comment that maps cannot be repeated therefore an early return is OK? (see rest of function body / lines below).

Reply:
I've restructured this so there is no longer an early return but have also added a comment about how maps are repeated fields in proto form but should not be rendered that way in the bones output.